pathfinding_service.rst

```diff
@@ -161,13 +177,16 @@ Example
         "error": "Invalid signature."
     }
 
-``api/1/paths``
-^^^^^^^^^^^^^^^
+``api/1/get_paths``
```

Comment:
why do we rename these?

Reply:
to make the naming consistent with the methods we implemented

Reply:
not sure about naming conventions, but I thought the get comes from `GET` verb in HTTP.

Reply:
Note: the `TokenNetwork` methods in the code are NOT the API interface. `TokenNetwork` is just the internal business logic.
The API interface that receives requests and converts them for the business logic is defined in `ServiceApi` and its registered `PathfinderResource` derivatives.

So that endpoint still is `'api/1/paths'`, called with a `GET`.

Reply:
What I described in the spec is exactly that internal logic, the api call is not interesting to spec out I guess. I'd prefer consistency with the internal method, which tells you where to look, if you want to find in the code for what is described in the spec.

Reply:
In that [part](https://raiden-network-specification.readthedocs.io/en/latest/pathfinding_service.html#public-endpoints) we explicitly define the REST API endpoints that will be used. So is shuld be `api/1/paths` here.

Reply:
Okay done. Please see if my additions to spec make sense to you then or if they should go somewhere else.